templates/components/element_detail_tabs.html

```diff
@@ -0,0 +1,458 @@
+{% extends "base.html" %}
+{% load humanize %}
+{% load guardian_tags %}
+{% load static %}
+{% load q %}
+
+{% block title %}
+{{ portfolio.title }} Component Detail
+{% endblock %}
+
+{% block head %}
+{% include "controls/_style-controls.html" %}
+<style>
+  .project-image {
+    width: 20px;
+    float: left;
+    margin: 4px 0 5px 8px;
+  }
+
+  .inline {
+    display: inline;
+  }
+
+  h3 {
+    color: #888;
+    #font-weight: bold;
+    margin-top: 20px;
+  }
+
+  #above-tab-content, #tab-content {
+    max-width: 950px;
+    margin: auto;
+  }
+
+  .control-text {
+    white-space: pre-wrap;
+    font-size: 11pt;
+    /*max-width: 700px;*/
+    font-family: trebuchet ms, sans-serif;
+    text-align: justify;
+    /*text-justify: inter-word;*/
+    line-height: 24px;
+  }
+
+  #control-lookup input {
+    width: 130px;
+  }
+
+  #combined_smt {
+    width: 80%;
+    /*margin: 12px 0 16px 0px;*/
+    padding: 8px;
+    /*display: none;*/
+  }
+
+  textarea {
+    overflow-y: scroll;
+    resize: vertical;
+  }
+
+  a {
+    color: #666;
+  }
+
+  #common-tab-count, #component-tab-count {
+    display: inline-block;
+    color: #666;
+    background-color: #dedede;
+    border-radius: 20px;
+    line-height: 1;
+    padding: 2px 6px;
+    font-size: 0.8em; font-weight: 600;
+  }
+
+  .panel-heading-smt, .panel-heading-smt-body {
+    white-space: pre-line;
+  }
+
+  .panel-heading a:hover {
+    text-decoration: none;
+    color: #000;
+  }
+
+</style>
```

Comment:
Will this add rework later for the work @kerryrm is doing? Is there somewhere else to put this?
Same comment throughout on inline styling.

Reply:
Agreed. I have been leaving the style tagged sections within the specific files for now.

Reply:
@omarabed15 I knew this would add to @kerryrm work, but wanted to avoid mixing work from different branches. Because this page's styles are identical to styles in some other pages, it seemed sensible to leave.